.circleci/config.yml

```diff
@@ -1,56 +1,47 @@
 # Javascript Node CircleCI 2.0 configuration file
 #
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
+
 version: 2
 
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:12
+    - image: circleci/node:15
```

Comment:
The change to the CI has nothing to do with the release :)

Reply:
But it worked out, I tried with this PR as a draft first :)